low-level/files/opt/seagate/sspl/setup/sspl_dev_deploy

```diff
@@ -403,8 +404,9 @@ class SSPLDevDeploy:
             "%s post_install --config yaml://%s" % (utils_setup_script, post_install_config),
             "%s post_install --config yaml://%s" % (sspl_setup_script, post_install_config),
             "%s prepare --config yaml://%s" % (sspl_setup_script, prepare_config),
-            "%s config --config yaml://%s" % (utils_setup_script, config_config),
+            "%s config --config yaml://%s" % (utils_setup_script, utils_config_tmpl),
```

Comment:
why this name change here. Similar names exists at other lines, signifying the mini provisioner stages i.e <mini-provi-stage>_config.
Now config_config is not proper i agree, though try to improve on that and not change the rule followed here.
Like can use 'miniprovConfig_config'

Reply:
@cdeshmukh it's not a name change, different conf file is parsed.
Earlier same config file was parsed to both utils_setup & sspl_setup script.
As per cortx-utils wiki, we require an input variable file to parsed in the config phase.
so I kept utils config input file inside the /tmp folder 

#L381
> utils_config_tmpl = "/tmp/utils.config.tmpl.%s" % (self.deploy_type)

Reply:
Below is the cortx-utils mini provisioning wifi ref:
https://github.com/Seagate/cortx-utils/wiki/%22cortx-py-utils%22-single-node-manual-provisioning

Reply:
oh i misunderstood, got it now. thanks